salt/kubernetes-master/cloud-config.jinja

```diff
@@ -0,0 +1,14 @@
+[Global]
+auth-url={{ pillar['cloud']['config']['auth_url'] }}
+username={{ pillar['cloud']['config']['username'] }}
+password={{ pillar['cloud']['config']['password'] }}
+tenant-name={{ pillar['cloud']['config']['project'] }}
+domain-name={{ pillar['cloud']['config']['domain'] }}
+region={{ pillar['cloud']['config']['region'] }}
+[LoadBalancer]
+lb-version=v2
+subnet-id={{ pillar['cloud']['config']['subnet'] }}
+create-monitor=yes
+monitor-delay=1m
+monitor-timeout=30s
+monitor-max-retries=3
```

Comment:
Theres also a really important option for cinder - since cinder more often than not lies!

    [BlockStorage]
    trust-device-path = false